src/Microsoft.Win32.Primitives/tests/Win32Exception.cs

```diff
@@ -56,11 +56,14 @@ public static void InstantiateExceptionWithLongErrorString()
             // and we can revise it accordingly.
 
             Win32Exception ex = new Win32Exception(0x268);
-            Assert.NotEqual("Unknown error (0x268)", ex.Message);
-            Assert.True(ex.Message.Length > 256);
+            if (ex.Message.Length > 256) // Message length for 0x268 is not > 256 characters in all cultures.
+            {
+                Assert.NotEqual("Unknown error (0x268)", ex.Message);
```

Comment:
No but this test will add no value in case the length is less than 256. The first test would have already tested it.